sdk/src/test/scala/com/horizen/WalletBoxSerializerTest.scala

```diff
@@ -30,7 +30,7 @@ class WalletBoxSerializerTest extends JUnitSuite with BoxFixture {
     // Test 1: serialization for core Box
     Random.nextBytes(transactionIdBytes)
     val walletBoxWithRegularBox = new WalletBox(
-      getRegularBox(getPrivateKey25519("seed1".getBytes), 1, 100),
+      getZenBox(getPrivateKey25519("seed1".getBytes), 1, 100),
```

Comment:
Consider to update _walletBoxWithRegularBox_ and _parsedWalletBoxWithRegularBox_ variable names.